src/libs/replaceSourceOrigin.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'underscore';
+import Config from '../CONFIG';
+
+/**
+ * Update the URL so images/files can be accessed depending on the config environment
+ *
+ * @param {String} urlString
+ * @returns {*|String}
```

Comment:
It looks like we always return a string, no?

```suggestion
 * @returns {String}
```